### DISCUSSION TITLE
What are the team’s goals for this product?

### DISCUSSION THREAD

**Discussion by JoBrad** [microsoft/winget-cli#317] [Q&A] [answered]
I’ll preface this by saying I’m definitely excited about and looking forward to a fully-featured official package manager for Windows. Major kudos to the team, and about damn time!
But herein lies the rub: is the goal of this project to simply provide a bare-bones app for accessing the official repository, or do you plan to make this a full-featured repo manager? Also, fully-featured will mean different things depending on your context and experience with other tools, so what is the team planning to do (and not planning to do) for this first phase, and into the next phase? I understand that everyone on this team is likely working on several projects at once, but it might be useful to know if a feature I have in mind is completely off the table due to project goals, or even if those goals haven’t yet been solidified beyond some base state.

**Comment by blogcraft** [comment]
I like where this product is going. But I would also like to know, is: How will Winget work alongside Windows Store? Are they going to be parallel ways to get software installed in Windows? Is the Store going to rely in Winget in the background? How is this relationship gonna work (if there is any at all)

**Comment by doctordns** [comment]
The relationship between Winget and WIndows store apps is interesting. And so is how this tool is ever going to be integratable with PowerShell and thus be useful in an enterprise environment?

**Comment by denelon** [comment, marked as answer]
We've put a roadmap in the repository at https://github.com/microsoft/winget-cli/blob/master/doc/windows-package-manager-v1-roadmap.md 

We decided to go open-source and release 0.1 as a preview so we could get more feedback from the community, and hopefully build a tool more people want to use.

Our next milestone will include support for installing store apps. The spec is in draft at https://github.com/microsoft/winget-cli/pull/367

There is also a request for native PowerShell support with community support (we're paying attention to 👍) at #221
